File: G_Projects/I_Vending_Machine/UtilityClasses/Item.java
package G_Projects.I_Vending_Machine.UtilityClasses;

import G_Projects.I_Vending_Machine.CommonEnums.ItemType;

public class Item {
    // private String id; //Should we keep an id?
    private ItemType type;
    private int price;

    public ItemType getType(){
        return type;
    }
    public void setType(ItemType type){
        this.type = type;
    }
    public int getPrice(){
        return price;
    }
    public void setPrice(int price){
        this.price = price;
    }
}

File: Python/A_Project/E_Vending_Machine/UtilityClasses/item.py
class Item:
    def __init__(self):
        self._item_type = None
        self._price = 0

    def get_type(self):
        return self._item_type

    def set_type(self, item_type):
        self._item_type = item_type

    def get_price(self) -> int:
        return self._price

    def set_price(self, price: int):
        self._price = price
